spec/features/visitors/a_visitor_gets_a_404_error_when_they_try_to_visit_project_new_page_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe 'As a guest' do
+  context 'When I try to visit new_project_path' do
+    scenario 'I see a 404' do
```

Comment:
I hate to be that guy but is this going to stay this way, or do we want to have it set to redirect to the sign up page?